clients/python-wrapper/lakefs/branch.py

```diff
@@ -124,6 +124,25 @@ def get_commit(self):
         self._commit = None
         return super().get_commit()
 
+    def cherry_pick(self, reference: ReferenceType, parent_number: Optional[int] = None) -> Commit:
```

Comment:
Can we add the force flag as well (with default to False)

Reply:
Sure. What does it actually do that a normal cherry-pick does not (there is no doc in the SDK argument)?

Reply:
@nicholasjng on second thought - please ignore this comment. This flag is related to a feature which is currently in development. We'll add it in the future if needed.

Reply:
So remove the `force` argument again?